main.js

```diff
@@ -67,4 +79,8 @@ const menubar = Menubar({
   ipcMain.on('app-ready', event => {
     event.sender.send('port-ready', port);
   });
+
+  ipcMain.on('error', function(e){	
+    console.log(e)
```

Comment:
And use `console.error` here

```suggestion
    console.error(event)
```